app/presenters/eastern_time_presenter.rb

```diff
@@ -0,0 +1,21 @@
+class EasternTimePresenter
+  def initialize(timestamp)
+    @timestamp = timestamp
+  end
+
+  def to_s
+    I18n.t(
+      'event_types.timestamp',
+      date: eastern_timestamp.strftime('%B %e, %Y'),
```

Comment:
Maybe a good use for `I18n.l`?

```yml
# some translation.yml
en:
  time:
    formats:
      date: '%B %e, %Y'
      time: '%-l:%M %p'
```

```ruby
I18n.t(
  'event_types.timestamp',
  date: I18n.lt(eastern_timestamp, format: :date),
  time: I18n.lt(eastern_timestamp, format: :time)
)
```

And I know on our ticket we specified full "Eastern" but I think we could get close with `"%z"`:

```yml
en:
  time:
    formats:
      event_timestamp: "%B %e, %Y at %-l:%M %p (%z)"
```

```ruby
I18n.t(eastern_timestamp, format: :event_timestamp)
```